app/src/main/java/com/lokico/PSWind/NOAA.java

```diff
@@ -1,17 +1,108 @@
 package com.lokico.PSWind;
 
+import android.graphics.Color;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
 
 public class NOAA extends AppCompatActivity {
 
+    private static final String TAG = "NOAA";
+    RequestQueue mRequestQueue;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_noa);
+
+        mRequestQueue = Volley.newRequestQueue(getApplicationContext());
```

Comment:
While it's relatively easy to understand, I'm a big stickler for using more comments than you're been using.  It's easier for other people to understand why, for example, you're setting to LOAD_NO_CACHE and enabling javascript if you explain yourself here.  It really helps people reviewing code.

Pretty frequently the "other people" I mentioned is actually "future you".  You'll look at something and ask "Who in their right mind wrote this code.  I have no idea what's going on here."  You go to look up who wrote it to ask them for an explanation ... and you're sad when you figure it out.  =)